### PR TITLE
2024.3.0 release

### PR DESCRIPTION
**Pull request by mryzhov**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set name = "openvino-tokenizers" %}
-{% set version = "2024.2.0" %}
+{% set version = "2024.3.0" %}
```
Comment (ilya-lavrenov) [inline recipe/meta.yaml:2]: you need to update OpenVINO version in .ci_support/migration files as well

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**